molecule/default/files/etc-nginx/conf.d/proxy-other.conf

```diff
@@ -42,6 +42,7 @@ server {
     }
 
     location /other/ {
+
```

Comment:
These seems to be moving newlines around. Is there a reason for it or can it be removed from this PR for clarity?

Reply:
It's to do with the formatting of the template. Adding newlines in the template improves its readability (I think it's fair to say it's difficult enough to read as it is), the downside is you end up with extra spaces. This is due to:
```
Line 1

{% if %}
New section
{% endif %}

Line 2
```
Becoming
```
Line 1


Line 3
```
when `New section` is omitted.

Reply:
Definitely :+1: for making the templates readable. I would not block this PR on these blank lines.
Moving forward, I would still vote for trying to minimize changes in generated configuration files if no variables are being changed, both to ease PR review and to prevent unnecessary server restarts.